frontend/app/src/people/personViewSlim.tsx

```diff
@@ -341,6 +386,37 @@ export default function PersonView(props: any) {
       return (
         <>
           <PageLoadSpinner show={loadingPerson} />
+          {/* <PersonIconButton
+                text={'Post a Bounty'}
+                endingIcon={'add'}
+                width={204}
+                height={48}
+                color={'success'}
+                style={{
+                  color: color.pureWhite,
+                  fontSize: '16px',
+                  fontWeight: '600',
+                  textDecoration: 'none'
+                }}
+                hoverColor={color.button_primary.hover}
+                activeColor={color.button_primary.active}
+                shadowColor={color.button_primary.shadow}
+                iconStyle={{
+                  fontSize: '16px',
+                  fontWeight: '400',
+                  top: '17px',
+                  right: '18px'
+                }}
+                onClick={() => {
+                  if (ui.meInfo && ui.meInfo?.owner_alias) {
+                    setShowFocusView(true);
+                  } 
+                  // else {
+                  //   showModal();
+                  // }
+                }}
+              />
+          <div>vdsvv</div> */}
```

Comment:
if we want to remove this we can just delete to keep the code clean, if some work needs to be done to use this lets add a TODO